pkg/arm/data/master-startup.sh

```diff
@@ -595,6 +595,7 @@ spec:
 EOF
 fi
 
+{{ if not DevelopmentMode }}
```

Comment:
`{{-` for all of these `if`s and `end`s

Reply:
I've also seen this in azuredeploy.json. What's the difference?

Reply:
eliminates whitespace